pkg/controller/config/process/excluder.go

```diff
@@ -95,8 +96,18 @@ func (s *Excluder) IsNamespaceExcluded(process Process, obj runtime.Object) (boo
 	}
 
 	if obj.GetObjectKind().GroupVersionKind().Kind == "Namespace" && obj.GetObjectKind().GroupVersionKind().Group == "" {
-		return s.excludedNamespaces[process][meta.GetName()], nil
+		return exactOrPrefixMatch(s.excludedNamespaces[process], meta.GetName()), nil
 	}
 
-	return s.excludedNamespaces[process][meta.GetNamespace()], nil
+	return exactOrPrefixMatch(s.excludedNamespaces[process], meta.GetNamespace()), nil
+}
+
+func exactOrPrefixMatch(boolMap map[util.PrefixWildcard]bool, ns string) bool {
+	for k, val := range boolMap {
+		if k.Matches(ns) {
+			return val
```

Comment:
nit: drop `val`, return an explicit `true`, `val` has no relation to whether a given field matches.

Reply:
If the `boolMap`'s values don't matter (i.e., we always return true), should it be refactored to be a map of `struct{}`?  If it's always returning `true`, then it's essentially functioning as a set.

Reply:
Yep, it's a set. `bool` is a little easier to work with for sets, as it allows you to test for membership just by accessing the value. Example: `isMember["a"]` returns `true` if `"a"` is a member, but the default value (`false`) if it's not.

Reply:
rodger that.  I've updated the function to always return `true`.